tests/gold_tests/autest-site/trafficserver.test.ext

```diff
@@ -403,6 +404,7 @@ def MakeATSProcess(
     p.Ready = When.FileContains(p.Disk.diags_log.AbsPath, "NOTE: Traffic Server is fully initialized")
 
     if select_ports:
+        uds_path = "/tmp/socket"
```

Comment:
`/tmp/socket` might be generic enough to accidentally clash with another socket. Let's put this in the sandbox.

My initial thought was to put the socket in the ts.Variables namespace and have the MakeCurlCommand reference that, but then we need to pass ts to each of those MakeCurlCommand calls which is kind of clunky. Let's instead put this in the `RunDirectory` which all the processes should be able to access:

So here in this trafficserver Process logic:

```python3
    uds_path = os.path.join(obj.RunDirectory, 'uds.socket')
    p.Variables.uds_path  = uds_path
```

Then the MakeCurlCommand logic can do the same type of thing:

```python3
    uds_path = os.path.join(p.RunDirectory, 'uds.socket')
    f'curl --unix-socket {uds_path} ' + cmd
```